<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionTagEditDialog</class>
 <widget class="QDialog" name="QuestionTagEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置标签</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="5">
    <widget class="QPushButton" name="delButton">
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="idLabel">
     <property name="text">
      <string>id:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QComboBox" name="tagComboBox"/>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QListView" name="listView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
